### DISCUSSION TITLE
Export vulnerabilities Docker to GoDojo

### DISCUSSION THREAD

**Discussion by pgabriel10** [DefectDojo/django-DefectDojo#6737] [General]
Hello Guys,

**Deployment method** *(select with an `X`)*
- [X] Docker Compose
- [ ] Kubernetes
- [X] GoDojo

I Installed the DefectDojo in Docker Compose, but now I discovery the another team use DefectDojo, but they use GoDojo.

So, I want use just 1 defect dojo for all vulnerabilites.

Is it possible export the vulnerabilities I had create in Docker version to GoDojo ?

**Comment by mtesauro** [comment]
Two suggestions:

(1) I'd move the data from the godojo install to the docker-compose install if it was me.  The docker-compose install is easier to update and has some processes automated (via the initializer container) that are manual with godojo.

(2) There's not a built-in way to move findings from one install to another. The easiest way to move those would be pull the findings out of the API from one install and push them using the API on the other install.  

Depending on if they product_types and products are the same between the two install, you may need to create product_types or products in the destination installation.  The same may be true for engagements and tests as well.  It really depends on how closely the two installs are setup/configured in terms of the data model (product_type, product, engagement, test, findings)
